src/py42/clients/devices.py

```diff
@@ -238,3 +238,31 @@ def get_settings(self, guid, keys=None):
         uri = u"/api/v4/device-setting/view"
         params = {u"guid": guid, u"keys": keys}
         return self._session.get(uri, params=params)
+
+    def get_agent_state(self, guid, propertyName):
```

Comment:
We use Hungarian case, `property_name`

Reply:
I've updated this in ebd235272b9b85156655f837475a2ec872406253